editor/styles/styles.scss

```diff
@@ -1,18 +1,9 @@
-$berta-bg-dark: #000;
-$berta-bg-mid-dark: #333;
-$berta-bg-light: #fff;
-$berta-nav-text: #b3b3b3;
-$berta-accent: #0c4dff;
-$berta-nav-text-active: #fff;
-
-$bt-header-font-size: 1rem;
-$bt-header-line-height: 1.375rem;
-$bt-header-item-padding: 1.1875rem 1.25rem;
-$bt-header-height: 3.75rem;
-
+@import 'variables';
```

Comment:
`styles` folder should go inside the `src` folder as it is a part of application source - `src/styles/[styles go here]`